sdk/eventhub/Azure.Management.EventHub/tests/EventHubManagementClientBase.cs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using Azure.Management.Resources;
+using Azure.Management.Resources.Models;
+using NUnit.Framework;
+
+namespace Azure.Management.EventHub.Tests
+{
+    [NonParallelizable]
+    public abstract class EventHubManagementClientBase : RecordedTestBase<EventhubTestEnvironment>
+    {
+        private const string ObjectIdKey = "ObjectId";
+        private const string ApplicationIdKey = "ApplicationId";
+
+        public string tenantId { get; set; }
```

Comment:
The first letter of all properties should be capital, i.e. `TenantId` instead of tenantId